### PR TITLE
Lesson 5: Ajax

### PR DESCRIPTION
**Pull request by seemcat**
Delete form & send data via XMLHttpRequest.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  <title></title>
+</head>
+<body>
+  Name: <input type = 'text' id = 'a1'/>
+  Comment: <input type = 'text' id = 'a2'/>
+  <input type = 'submit' id = 'b1'>
+<script>
+  const button = document.getElementById('b1');
+  button.onclick = () => {
+    const name = document.getElementById('a1').value;
```
Comment (songz) [inline public/index.html]: better variable naming please. `a1`, `b1`, etc.

File: public/xmlPublicHTML.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Name: undefined Comment: undefined
+Name: undefined Comment: undefined
```
Comment (songz) [inline public/xmlPublicHTML.txt]: plz don't commit generated files.

File: publicHTML.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const fs = require('fs');
+const app = express();
+
+app.listen(3002);
+app.use(express.static('public'));
+
+/*
```
Comment (songz) [inline publicHTML.js]: y did you comment this out?

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title></title>
+</head>
+<body>
+  Name: <input type = 'text' id = 'a1'/>
+  Comment: <input type = 'text' id = 'a2'/>
```
Comment (saumya665) [inline public/index.html]: better variable names please for the id's!

File: public/index.html [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  <input type = 'submit' id = 'b1'>
+<script>
+  const button = document.getElementById('b1');
+  button.onclick = () => {
+    const name = document.getElementById('a1').value;
+    const comment = document.getElementById('a2').value;
+    const xhttp = new XMLHttpRequest();
+    xhttp.open('GET', `/XMLHttpRequest?a1=${name}&a2=${comment}`);
```
Comment (saumya665) [inline public/index.html]: use better variable names in the url as well

File: public/index.html (hunk 1 above)
Comment (seemcat) [reply public/index.html]: Done!

File: public/xmlPublicHTML.txt (hunk 2 above)
Comment (seemcat) [reply public/xmlPublicHTML.txt]: Done!

File: publicHTML.js (hunk 3 above)
Comment (seemcat) [reply publicHTML.js]: It's gone!

File: public/index.html (hunk 5 above)
Comment (seemcat) [reply public/index.html]: Yes, I agree 😃

**Review by songz — APPROVED** [review]

**Review by MadhuBeesetty — APPROVED** [review]

**Pull request merged**